src/processes.c

```diff
@@ -866,15 +878,23 @@ static void ps_submit_proc_stats (
     {
         vl.values[0].derive = io_rchar;
         vl.values[1].derive = io_wchar;
-        dispatch_value_helper(&vl, "ps_disk_octets", NULL, 2, doing_detail,
+        dispatch_value_helper(&vl, "io_octets", NULL, 2, doing_detail,
                 config->ps_disk_octets);
     }
 
     if ( (io_syscr != -1) && (io_syscw != -1) )
     {
         vl.values[0].derive = io_syscr;
         vl.values[1].derive = io_syscw;
-        dispatch_value_helper(&vl, "ps_disk_ops", NULL, 2, doing_detail,
+        dispatch_value_helper(&vl, "io_ops", NULL, 2, doing_detail,
+                              config->ps_disk_ops);
+    }
+
+    if ( (io_diskr != -1) && (io_diskw != -1) )
+    {
+        vl.values[0].derive = io_diskr;
+        vl.values[1].derive = io_diskw;
+        dispatch_value_helper(&vl, "disk_octets", NULL, 2, doing_detail,
```

Comment:
Should this not be `ps_disk_octets`? And shouldn't it be guarded by `config->ps_disk_octets`?
The issue with keeping the existing name is that the metric value range will suddenly jump. The problem with the new name is that it needs a new mapping. Let's figure out which is the lesser evil.

Reply:
Have you seen the comments in the change description? I'm doing the same as the upstream patch, which we agreed to backport. Upstream decided to use a new name. I think the bigger evil would be to make our semantic change that's different from upstream's.

Reply:
Right, it does make sense to stick to the upstream implementation. The second question, however, applies specifically to this port (that issue is not present in the upstream codebase anymore). I think `config->ps_disk_ops` is a typo, and it should be `config->ps_disk_octets` (unless I'm missing something).

Reply:
I see. The latest version of this plugin doesn't seem to have want_detail_configuration_t or anything like it, as far as I can tell. I changed it to ps_disk_octets, but it probably won't matter.